README.md

```diff
@@ -44,20 +35,14 @@ We have identified several **defining characteristics** about each role at Octop
 
 Rate yourself at the **defining characteristics** level so it's not too hard to maintain over time; the examples act as a frame of reference for the rating.
 
-If you're aiming for a promotion or a different role, rate yourself on both your current role and your desired role. Looking at the desired role will provide clarity about gaps you might need to fill or grow into. You should discuss your desire before rating with a peer or mentor, so that they can provide feedback on your goals.
+If you're aiming for a promotion or a different role, rate yourself on both your current role and your desired role. Looking at the desired role will provide clarity about gaps you might need to fill or grow into. You should discuss your appetite with your manager so they can provide feedback on your goals.
 
 Let's use simple terms for the ratings:
 
-- `No brainer`: _"You're great at this! We don't even need to talk about it."_
-- `Maturing`: _"Here's some suggestions, take them or leave them - it's your call."_
-  - It makes sense to record the suggestions.
-- `Needs coaching`: _"This is an area you're weak in, let's work out a plan and walk this out together."_
-  - It makes sense to record the coaching plan.
-
-Let's call special attention to clearly outstanding/exemplary behaviour:
-
-- `Outstanding`: _You simply shine in this area like very few others. I think people deserve to know this about you._
-  - This is more than just saying "I'm really, really good at this". It's saying that you're right up there with the best few across all of Octopus at this.
+- `Performing`: _"You're great at this! You're doing exactly what we expect of someone in this role and level. Keep up the great work!"_
+- `Exceeding`: _"In this area, you are exceeding what we expect of someone in this role and level."_
+- `Maturing`: _"You've got some growth required in this area, but the trajectory is good. Here's some suggestions, take them or leave them - it's your call."_
+- `Coaching needed`: _"You've got some growth required in this area and the trajectory isn't good. Let's work out a coaching plan and turn this around together."_
```

Comment:
I've typically seen us associate the words "coaching" and "supporting" _within_ the maturing role (from the Situation Leadership model), and words such as "improvement needed" to indicate that there's a clear problem/gap that needs addressing and the trajectory is not good. I.e. "Improvement needed" is effectively the signal before more drastic measures or harder conversations need to be considered (early signal towards either performance management 😞  or improvement taken 😌 ).

Is there a risk that "coaching" becomes associated with "my trajectory is not good"?

If so, an alternative that aligns with the HiBob/SI ratings we've given in the past would be "Improvement needed". And perhaps some changes to the maturing above to clarify a spectrum:

E.g.
> ... Here's some suggestions, take them or leave them - it's your call; Here's some coaching we'll work through together to help ensure you are setup to succeed on your maturing journey."_

Reply:
Tough one. With the SL model, I can also see how coaching implies they are in that "warning" state, so perhaps it is a good signal-term that the trajectory is not (yet) good 👍

Reply:
Good call @MarkSiedle . Can you please make a suggestion to this file?